tokenomics/etherdai.md

```diff
@@ -0,0 +1,33 @@
+# EtherDai
+
+EtherDai is a synthetic Ether token backed by liquid staked Ether tokens and debt issued by Maker. It is pegged to the value of Ether. EtherDai is one of the central products of the Endgame Plan. 
```

Comment:
The sentence is slightly confusing to me. Is EtherDAI issued by Maker or the debt issued by Maker?